tmpl/package.tmpl

```diff
@@ -107,6 +107,24 @@ func {{.GoName}}({{template "paramsGoDecl" .Parameters}}){{if not .Return.IsVoid
 }
 {{end}}
 
+// Init initializes the OpenGL bindings by loading the function pointers (for
+// each OpenGL function) from the active OpenGL context.
+//
+// It must be called under the presence of an active OpenGL context, e.g. always
+// after calling window.MakeContextCurrent() and always before calling any
+// OpenGL functions exported by this package.
+//
+// On Windows, Init loads pointers that are context-specific (and hence you
+// must re-init if switching between OpenGL contexts, although it may work if
+// the contexts belong to the same graphics driver/device).
+//
+// On OS X and Linux, the behavior also different, but code written compatible
+// with the Windows behavior is compatible with OS X and Linux. That is, always
+// Init under an active OpenGL context, and always re-init after switching
+// graphics contexts.
+//
+// For information about caveats of Init, you should read the "Platform Specific
+// Function Retrieval" section of https://www.opengl.org/wiki/Load_OpenGL_Functions
```

Comment:
I would still put a period at the end of this sentence, even though it ends with a URL. I think most people and parsers can handle such case.